SurveyTests/Sources/Data/Repositories/SurveyRepositorySpec.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  SurveyRepositorySpec.swift
+//  Survey
+//
+//  Created by Khanh on 28/11/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+//  swiftlint:disable closure_body_length
+
+import Combine
+import Nimble
+import Quick
+import SwiftUI
+
+@testable import Survey
+
+final class SurveyRepositorySpec: QuickSpec {
+
+    override func spec() {
+        var surveyRepository: SurveyRepository!
+        var networkAPIMock: NetworkAPIMock!
+
+        describe("a SurveyRepository") {
+
+            beforeEach {
+                networkAPIMock = NetworkAPIMock()
+                surveyRepository = SurveyRepository(api: networkAPIMock)
+            }
+
+            context("when getSurveyList is called") {
```

Comment:
this `context` should be `describe`
```suggestion
            describe("its getSurveyList") {
```